server.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+let express = require('express');
+let app = express();
+
+app.use(express.static(__dirname + '/client'));
+
+app.listen(8888, function(){
```

Comment:
I think you want a space after the `function()` and before the `{`:

```
app.listen(8888, function() {
```

Or however you like it, but be consistent.